docs/querying/functions.md

```
@@ -891,6 +891,36 @@ queries, this returns `0`.
+Very small non-zero slopes can return very large ETAs, matching the linear
+model.
+
+The function should only be used with gauges and only works for float samples.
+Elements in the range vector that contain only histogram samples are ignored
+entirely. For elements that contain a mix of float and histogram samples, only
+the float samples are used as input, which is flagged by an info-level
+annotation.
+
+For example, to flag disks predicted to fill within the next 24 hours:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)